YogaKit/Source/YGLayout.m

```diff
@@ -21,26 +21,30 @@ - (void)set##capitalized_name:(type)lowercased_name             \
   YGNodeStyleSet##capitalized_name(self.node, lowercased_name); \
 }
 
-#define YG_VALUE_PROPERTY(lowercased_name, capitalized_name)    \
-- (CGFloat)lowercased_name                                      \
-{                                                               \
-  YGValue value = YGNodeStyleGet##capitalized_name(self.node);  \
-  if (value.unit == YGUnitPoint) {                              \
-    return value.value;                                         \
-  } else {                                                      \
-    return YGUndefined;                                         \
-  }                                                             \
-}                                                               \
-                                                                \
-- (void)set##capitalized_name:(CGFloat)lowercased_name          \
-{                                                               \
-  YGNodeStyleSet##capitalized_name(self.node, lowercased_name); \
+#define YG_VALUE_PROPERTY(lowercased_name, capitalized_name)                       \
```

Comment:
Once this lands, I really want to remove these defines. It might be a bit of extra code, but I'd rather try and use function pointers instead of this.